mitodl/mit-open
# Merge `main` into `nextjs`

### What are the relevant tickets?
For https://github.com/mitodl/hq/issues/5395

### Description (What does it do?)
Merges `main` into `nextjs` and fixes a few issues that arose.

### How can this be tested?
Run locally. You should get recent changes from `main`. Notably:
- header
- Ecommerce page

<!--- Uncomment and add steps to be completed before merging this PR if necessary
### Checklist:
- [ ] Migrate ecommerce page. I forgot 😦
